docs/source/app-dev/parties-users.rst

```diff
@@ -0,0 +1,88 @@
+.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Parties and Users On the Daml Ledger
```

Comment:
Maybe "Parties and Users on _a_ Daml ledger"
Since there isn't one main Daml ledger...